dump-event-schema-compiler-plugin/src/test/resources/StateData.scala

```diff
@@ -0,0 +1,10 @@
+package org.random.project
+
+import akka.persistence.typed.scaladsl.Effect
+
+object StateData {
```

Comment:
Given that now state (and not only events) are handled by the plugin... WDYT about rename to e.g. `dump persistence schema`?